roundabout-core/src/main/java/at/fhv/itm3/s2/roundabout/util/dto/In.java

```diff
@@ -0,0 +1,13 @@
+package at.fhv.itm3.s2.roundabout.util.dto;
+
+public class In {
```

Comment:
I don't know where this comes from, but the "In" and "Out" are not very tied to a "probability"-property.
What I ment, that there is no connection in the naming and the content of the class. 
Can't help with another name, as I do not fully understand the meaning of the "In" and "Out" class - yet.

Reply:
These are JAXB DTO classes for loading the XML config, so the naming is like the XML names. We don't work with these, they are just DTOs.